release_notes/migrating_to_0_20.md

```diff
@@ -0,0 +1,41 @@
+# NGS CLI v 0.20.0
+
+Version 0.20 brings compatibility with JWT v2 functionality added to the NATS ecosystem. Future updates to NGS will introduce features like JetStream which will require version 2 if not deployed as a leaf node.
+
+NGS is fully compatible with JWT v1 or v2 projects. However, the tooling that is used to manage your account (`ngs` cli) and your configurations (`nsc`) determines the version `ngs` cli that you can use, so you should update now, simply to have it at the current version.
+
+However, by default the `ngs` cli installer will default to `nsc` at its latest version. If you are starting a new project, stop reading as the tooling will be fully compatible.
+
```

Comment:
The `ngs` cli installer will default to `nsc` at its latest version, so if you are starting a new project, you won't need to review the rest of this document as the tooling will be fully compatible.